src/main/java/com/faforever/client/chat/ChatUserControl.java

```diff
@@ -76,17 +115,115 @@ public PlayerInfoBean getPlayerInfoBean() {
   public void setPlayerInfoBean(PlayerInfoBean playerInfoBean) {
     this.playerInfoBean = playerInfoBean;
 
+    configureColor();
+    addPrettyColorListener();
     configureCountryImageView();
     configureAvatarImageView();
     configureClanLabel();
+    configureGameStatusView();
+    configureRatingTooltip();
 
     usernameLabel.setText(playerInfoBean.getUsername());
   }
 
+  private void configureRatingTooltip() {
+    playerInfoBean.chatOnlyProperty().addListener((observable1, oldValue1, newValue1) -> {
+      if (!newValue1) {
+        Tooltip userRatingTooltip = new Tooltip(playerInfoBean.getCountry());
+
+        String globalRating = i18n.get("playerRatingFormat.global", RatingUtil.getGlobalRating(playerInfoBean));
+        String ladderRating = i18n.get("playerRatingFormat.ladder", RatingUtil.getLadderRating(playerInfoBean));
+
+        userRatingTooltip.setText(String.format("%s \n %s", globalRating, ladderRating));
```

Comment:
Instead of defining two string formats and then joining them together (with unnecessary spaces :-)) we better just create one string format already containing the newline. `playerRatingFormat=Global Rating: {0,number,#}\nLadder Rating: {0,number,#}